Cargo.toml

```
@@ -12,16 +12,18 @@ readme = "README.md"
 include = ["src/**/*", "README.md", "CHANGELOG.md"]
 
 [dependencies]
-mongodb = { version = "^1.2", default-features = false }
+mongodb = { version = "2.0.2", default-features = false }
```

```suggestion
mongodb = { version = "2", default-features = false }
```

"2" for `>=2.0.0 && <3.0.0`